tasks/Bartender.tex

```diff
@@ -0,0 +1,29 @@
+\section{Bartender [Party Host]}
+The home owner will teach the robot to make a drink which guests at the party really like!
+
+\subsection{Focus}
+Learning from demonstration.
+
+\subsection{Main Goal}
+The home owner will teach the robot using demonstrations how to make a drink. The robot is then to repeat the task. A designated team member will show the home owner how to perform the demonstration. These drinks need to be \textit{good}, meaning that the robot must pour each mixer in for the right amount of time to get the right amount of mixer in.
```

Comment:
Getting a working blender in the arena and liquids might be problematic. Also the whole mixed drinks things in case RoboCup ever goes to a location which is dry... but the concept of the task, learning and programming by demonstration is very nice.

Reply:
Yeah. We were thinking that the robot would pick up bottles and pretend to mix drinks, rather than actually pour liquids, which is what we do in similar research activities at UT. Well, we don't use alcohol in those activities, but the idea holds.

Reply:
No liquids because it's smoothies now, and picking up fruit? Maybe this is still good?